qualtran/bloqs/gf_arithmetic/gf2_inverse_test.py

```diff
@@ -0,0 +1,55 @@
+#  Copyright 2024 Google LLC
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import pytest
+import sympy
+from galois import GF
+
+from qualtran.bloqs.gf_arithmetic.gf2_inverse import (
+    _gf2_inverse_symbolic,
+    _gf16_inverse,
+    GF2Inverse,
+)
+from qualtran.resource_counting import get_cost_value, QECGatesCost, QubitCount
+from qualtran.testing import assert_consistent_classical_action
+
+
+def test_gf16_multiplication(bloq_autotester):
+    bloq_autotester(_gf16_inverse)
+
+
+def test_gf2_multiplication_symbolic(bloq_autotester):
+    bloq_autotester(_gf2_inverse_symbolic)
+
+
+def test_gf2_multiplication_symbolic_toffoli_complexity():
+    bloq = _gf2_inverse_symbolic.make()
+    m = bloq.bitsize
+    assert get_cost_value(bloq, QECGatesCost()).toffoli - m**2 * (m - 2) == 0
```

Comment:
is the cost just toffolis? if so,  then in other tests we have been calling `total_t_and_ccz_count` and assertign that `n_t` == 0 and then checking `n_ccz` == the symbolic cost

Reply:
Used `total_toffoli_only` which implicitly asserts that `n_t == 0`